revscoring/features/diff.py

```diff
@@ -31,17 +31,73 @@ def process_chars_added(diff_added_segments):
 
 chars_added = Feature("diff.chars_added", process_chars_added,
                       returns=int, depends_on=[diff.added_segments])
+"""
+Represents number of characters added in this edit.
+
+:Returns:
+    int
+
+:Example:
+    ..code_block :: python
+
+        >>> from revscoring.features import diff
+        >>> list(extractor.extract(655097130, [diff.chars_added]))
+        [297]
+"""
 
 def process_chars_removed(diff_removed_segments):
     return len("".join(diff_removed_segments))
 
 chars_removed = Feature("diff.chars_removed", process_chars_removed,
                         returns=int, depends_on=[diff.removed_segments])
+"""
+Represents number of characters removed in this edit.
+
+:Returns:
+    int
+
+:Example:
+    ..code_block :: python
+
+        >>> from revscoring.features import diff
+        >>> list(extractor.extract(655097130, [diff.chars_removed]))
+        [297]
+"""
 
 proportion_of_chars_removed = chars_removed / \
               modifiers.max(parent_revision.chars, 1)
+"""
+Represents ratio of characters removed in this edit compared to overall
+characters in parent revision.
+
+:Returns:
+    float
+
+:Example:
+    ..code_block :: python
+
+        >>> from revscoring.features import diff
+        >>> list(
+            extractor.extract(609079959, [diff.proportion_of_chars_removed]))
+        [1.0]
+"""
 proportion_of_chars_added = chars_removed / \
             modifiers.max(revision.chars, 1)
+"""
+Represents ratio of characters removed in this edit compared to overall
+characters in revision.
+
+:Returns:
+    float
+
+:Example:
+    ..code_block :: python
+
+        >>> from revscoring.features import diff
+        >>> list(
+            extractor.extract(609079959, [diff.proportion_of_chars_added]))
```

Comment:
The feature is obviously wrong [1] but I can't understand intention of that feature so I can't fix it.

[1] proportion_of_chars_*added* = chars_*removed* / \